bin/compileI18n.js

```diff
@@ -0,0 +1,7 @@
+var fs = require('fs');
+var MessageFormat = require('messageformat');
+
+const data = fs.readFileSync('i18n/oceans.json');
+const mf = new MessageFormat('en');
+const i18n = mf.precompileObject(JSON.parse(data));
+fs.writeFileSync('i18n/oceans.js', 'export const oceans_locale = ' + i18n);
```

Comment:
should we prepend a comment when we write to `i18n/oceans.js` that explains that it's a generated file and shouldn't be edited manually?